src/pybind/mgr/dashboard/openapi.yaml

```diff
@@ -6060,6 +6060,28 @@ paths:
       summary: Sets OSD flags for a subset of individual OSDs.
       tags:
       - OSD
+  /api/osd/dump:
+    get:
+      parameters: []
+      responses:
+        '200':
+          content:
+            application/vnd.ceph.api.v1.0+json:
```

Comment:
AFAICR we agreed on marking new endpoints as experimental by default before promoting them to stable.
@pereman2 @nizamial09 @avanthakkar @Waadkh7  Thoughts ?

Reply:
I think that was the agreement.